Readme.md

```diff
@@ -35,18 +35,26 @@ An important property of CliFx, when compared to some other libraries, is that i
 
 ## Usage
 
-- [Quick start](#quick-start)
-- [Binding arguments](#binding-arguments)
-- [Argument syntax](#argument-syntax)
-- [Value conversion](#value-conversion)
-- [Multiple commands](#multiple-commands)
-- [Reporting errors](#reporting-errors)
-- [Graceful cancellation](#graceful-cancellation)
-- [Dependency injection](#dependency-injection)
-- [Testing](#testing)
-- [Debug and preview mode](#debug-and-preview-mode)
-- [Reporting progress](#reporting-progress)
-- [Environment variables](#environment-variables)
+- [CliFx](#clifx)
+  - [Download](#download)
+  - [Features](#features)
+  - [Screenshots](#screenshots)
+  - [Usage](#usage)
+    - [Quick start](#quick-start)
+    - [Binding arguments](#binding-arguments)
+    - [Argument syntax](#argument-syntax)
+    - [Value conversion](#value-conversion)
+    - [Multiple commands](#multiple-commands)
+    - [Reporting errors](#reporting-errors)
+        - [Customizing Error Reporting](#customizing-error-reporting)
+    - [Graceful cancellation](#graceful-cancellation)
+    - [Dependency injection](#dependency-injection)
+    - [Testing](#testing)
+    - [Debug and preview mode](#debug-and-preview-mode)
+    - [Reporting progress](#reporting-progress)
+    - [Environment variables](#environment-variables)
+  - [Benchmarks](#benchmarks)
+  - [Etymology](#etymology)
```

Comment:
Let's keep the table of contents only for usage

Reply:
Oops,  I think VSCode auto generated the whole table when I saved after adding my section. I'll fix.